CHANGELOG.md

```diff
@@ -48,6 +48,9 @@ All notable, unreleased changes to this project will be documented in this file.
 - Change metadata mutations to use token for order and checkout as identifier - #8426 by @IKarbowiak
   - After changes, using the order `id` for changing order metadata is deprecated
 - Add `withChoices` flag for Attribute type - #7733 by @dexon44
+`CheckoutAddPromoCode`, `CheckoutPaymentCreate` will raise a ValidationError when product in the checkout is
+unavailable - #8978 by @IKarbowiak
+- Fix crash when Avalara plugin was used together with Webhooks plugin for shipping methods - #9121 by @rafalp
```

Comment:
If I understand correctly, this PR changes the business logic behind calculating available shipping methods. Previously, I think always the gross amount was used? Now it uses the "base price" which can mean net or gross according to SiteSettings.

I think it would be worth to indicate the change in behaviour in the changelog on top of fixing the infinite recursion error.

Reply:
@stnatic this if fair point, I'll add this to changelog.